distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1618,6 +1658,172 @@ public void testFederatedDownloadProductToCacheOnlyCacheDisabled() throws Except
                 .resolve(CSW_SOURCE_ID + "-" + metacardId + ".ser")), is(false));
     }
 
+    @Test
+    public void testTwoUsersSameProductRetrySuccess() throws Exception {
+
+        String filename = "product2.txt";
+
+        CometDClient cometDClient1 = setupCometDClient(Arrays.asList(NOTIFICATIONS_CHANNEL,
+                ACTIVITIES_CHANNEL));
+        CometDClient cometDClient2 = setupCometDClient(Arrays.asList(NOTIFICATIONS_CHANNEL,
+                ACTIVITIES_CHANNEL));
+
+        setupCswServerForSuccess(cometDClient1, filename);
+        setupCswServerForSuccess(cometDClient2, filename);
+
+        // where 3 is the number of expectations and 9 is the number of activities
+        checkExpectations(cometDClient1, 3, 9);
+        checkExpectations(cometDClient2, 3, 9);
```

Comment:
my options were explaining them or creating an const, since this is local to the function, it doesn't make any sense to create a constant, so i made the comment as per the earlier comment